### ISSUE TITLE
growl & syntax errors etc

### ISSUE THREAD

**Issue by j0hnsmith** [less/less.ruby#119]
I would be nice if using
    lessc -gw ...

or any combination of watch and growl, that you didn't have to Press [return] to continue... when there's a syntax error as growl already notifies the user. This way you could just fix the error and continue.

**Comment by guns** [comment]
I agree that just fixing the error and resaving is more convenient than having to press `return`. I logged a similar issue here:

http://github.com/cloudhead/less/issues/#issue/116

and a proposed fix here:

http://github.com/guns/less/tree/guns-robust-watch

**Comment by cloudhead** [comment]
I was thinking about this too earlier, will see what I can do! (also going to merge in your patch at some point gun)

**Comment by DouweM** [comment]
This would be nice also because it notifies you when a file that's being `--watch`ed has been compiled again. Currently, I don't know for sure if it has been compiled since the last time, as I'm not looking at my terminal all the time, to notice the extra line that appears... I just wait a couple of seconds to see if a new line pops up, and if it doesn't, I suppose it already popped up before I started looking.
